docs/book/src/reference/glossary.md

```diff
@@ -5,7 +5,7 @@
 # A
 ---
 
-### Add-ons
```

Comment:
I don't have historical context here, but might be we are using h3 intentionally in order to keep the page layout a little bit more concise

Reply:
This is one of the things mdlint complains about - going from h1 to h3. But really no reason we can't for formatting reasons, especially since we aren't generating any TOCs based off of this. I will switch these back.